modules/op-creating-mounting-kms-authentication-token-secret.adoc

```diff
@@ -0,0 +1,56 @@
+// This module is included in the following assemblies:
+// * secure/using-tekton-chains-for-openshift-pipelines-supply-chain-security.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="creating-mounting-kms-authentication-token-secret_{context}"]
+= Creating and mounting the KMS authentication token secret
+
+You can provide the authentication token for the KMS server (for example, the the value of `VAULT_TOKEN` if the KMS provider is Hashicorp Vault) using a secret. You must create this secret, mount it on the {tekton-chains} controller, and set the `signers.kms.auth.token-dir` parameter to the directory where the secret is mounted.
+
+.Prerequisites
+
+* You installed the `oc` command-line utility.
+* You are logged in to your {OCP} cluster with administrative rights for the `tekton-chains` namespace.
+
+.Procedure
+
+. Create a secret named `kms-secrets` with the `KMS_AUTH_TOKEN` file that contains the authentication token for the KMS server by entering the following command:
```

Comment:
Same as above
User can create secret with any key now, we can put this as an example